misk-aws/src/main/kotlin/misk/jobqueue/sqs/ResolvedQueue.kt

```diff
@@ -13,7 +13,8 @@ internal class ResolvedQueue(
   val url: String,
   val region: AwsRegion,
   val accountId: AwsAccountId,
-  val client: AmazonSQS
+  val client: AmazonSQS,
+  val maxRetries: Int,
```

Comment:
Similarly `maxRetries` is a bit of a misnomer, that is controlled by the `max_delivery_attempts` setting on the queue, but this is just setting a ceiling on the backoff time

Reply:
maxRetries is essentially the max_delivery_attempts that we retrieve from the RedrivePolicy though or am i missing some context? 🤔